platform/default/thread_local.cpp

```diff
@@ -58,8 +58,8 @@ void ThreadLocal<T>::set(T* ptr) {
     }
 }
 
-template class ThreadLocal<RunLoop>;
```

Comment:
The code that used the template was in `RunLoop`, but has since been removed in favor of a `ThreadLocal` on `Scheduler`